.github/workflows/pr-quality.yml

```
@@ -0,0 +1,210 @@
+          echo "deadcode_reduction=${DEADCODE_REDUCTION}" >> $GITHUB_OUTPUT
+          echo "gates_passed=${GATES_PASSED}" >> $GITHUB_OUTPUT
+          echo "gates_failed=${GATES_FAILED}" >> $GITHUB_OUTPUT
+          echo "gates_warnings=${GATES_WARNINGS}" >> $GITHUB_OUTPUT
+          echo "overall_status=${STATUS}" >> $GITHUB_OUTPUT
+
+      - name: Extract top warning types
+        id: warnings
+        run: |
+          # Extract top 5 warning types in a simpler format
+          TOP_WARNINGS=$(grep -A 10 "=== Top.*Warning Types ===" quality-full.txt | tail -n +2 | head -n 5 | sed 's/^[[:space:]]*//' | sed 's/[[:space:]]*$//')
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)